main/mafia.hs

```diff
@@ -327,20 +328,65 @@ mkFileMap = Map.fromList . fmap (\path -> (takeFileName path, path))
 putUpdateReason :: CacheUpdate -> EitherT MafiaViolation IO ()
 putUpdateReason sync =
   case sync of
-    Delete file
-     -> do liftIO (T.hPutStrLn stderr ("Cache: Removed " <> takeFileName file))
+    Delete file -> do
+      liftIO (T.hPutStrLn stderr ("Cache: Removed " <> takeFileName file))
 
-    Update src _
-     -> do rel <- fromMaybe src <$> makeRelativeToCurrentDirectory src
-           liftIO (T.hPutStrLn stderr ("Cache: Modified " <> rel))
+    Update src _ -> do
+      rel <- fromMaybe src <$> makeRelativeToCurrentDirectory src
+      liftIO (T.hPutStrLn stderr ("Cache: Modified " <> rel))
```

Comment:
^ sorry this is just a style fixup so all the case expressions are the same in here